test/tomo/operators/ray_trafo_test.py

```diff
@@ -141,7 +143,7 @@ def projector(request):
     elif geom == 'helical':
         # Discrete reconstruction space
         reco_space = odl.uniform_discr([-20, -20, 0], [20, 20, 40],
-                                       [n_voxels] * 3, dtype='float32')
+                                       [n_voxels] * 3, dtype=dtype)
```

Comment:
Actually `n_voxels` is a misleading name since it reads as "total number of voxels". It should be changed to something else, just `n` should be fine for tests.